### PR TITLE
Converted svg_to_png to a class

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes : #668

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

-
- Converted `svg_to_png` to a class

### CODE REVIEW

**Comment by yashLadha** [comment]
Please review @djmgit @ParthS007 @vaibhavsingh97 @gabru-md

File: backend/app/svg_to_png.py [hunk 1]
```diff
@@ -3,71 +3,74 @@
...
+            text_nodes = path.getchildren()
+            path.set("style", style_detail)
+            for t in text_nodes:
+                text_style_detail = t.get("style")
+                text_style_detail = text_style_detail.split(";")
+                text_style_detail[-1] = "fill:" + str(fill)
+                text_style_detail = ";".join(text_style_detail)
+                t.set("style", text_style_detail)
+        etree.ElementTree(element).write(filename, pretty_print=True)
+        print("Text Fill saved!")
```
Comment (djmgit) [inline backend/app/svg_to_png.py]: Add end of file line, rest looks fine :+1:

Comment (yashLadha) [reply backend/app/svg_to_png.py]: Done @djmgit :+1:

Comment (yashLadha) [reply backend/app/svg_to_png.py]: done :+1:

Comment (yashLadha) [reply backend/app/svg_to_png.py]: done :+1:

File: backend/app/svg_to_png.py [hunk 2]
```diff
@@ -3,71 +3,75 @@
...
+
```
[the remark this answers is not in the document]
Comment (yashLadha) [reply backend/app/svg_to_png.py]: This is necessary LOL :stuck_out_tongue:

**Comment by yashLadha** [comment]
Please review again @djmgit

**Comment by yashLadha** [comment]
Please review @djmgit

**Comment by yashLadha** [comment]
Please review @gabru-md @ParthS007 .

**Pull request merged**